spec/models/user_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user) { Fabricate(:user) }
```

Comment:
Same as the comment below

https://github.com/blyscuit/rails-ic/pull/32/files#r1217352048